openapi.yaml

```diff
@@ -1521,6 +1585,25 @@ components:
           description: The fully-qualified property name, including the 'namespace'.
         value:
           type: string
+    GalasaProperty:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          properties:
+            namespace:
+              type: string
+              description: The namespace the property resides in
+            name:
+              type: string
+              description: The name of the property
```

Comment:
Need to say what valid characters are, a-z A-Z _ - . 
Need to say that there must be at least 2 parts to the property, separated by a '.' character. Parts of a property must not be blank.

Reply:
Is it case sensitive ?